README.md

```diff
@@ -108,7 +114,16 @@ type versionMessage struct {
 Encoded in the usual manner.
 Both sides verify that the version matches.
 If there is a mismatch, the connection is closed.
-The current version of the API, as described here, is `1`.
+The current version of the API, as described here, is `2`.
+
+### Changelog
+
+Version 1 is the initial version of the format.
+It contains the framed messages, requests, and responses.
+
+Version 2 introduces block presences to pushed Bitswap messages.
```

Comment:
Link to [block presences](https://github.com/ipfs/go-bitswap/blob/master/docs/how-bitswap-works.md)?

Reply:
Oh wow, I did not know that existed, thanks!